lib/frames/frame.js

```diff
@@ -179,8 +179,11 @@ AMQPFrame.prototype._writePayload = function(bufBuilder, options) {
   codec.encode(performative, bufBuilder);
   var payload = this._getAdditionalPayload();
   if (payload !== undefined) {
-    //debugAMQP('Encoding payload: ' + JSON.stringify(payload));
-    codec.encode(payload, bufBuilder);
+    if (payload instanceof Buffer) {
```

Comment:
is this check relevant anymore now that we basically assume `Buffer`s for receive and send?